server.js

```diff
@@ -0,0 +1,59 @@
+const http = require('http');
+const fs = require('fs');
+//const public = require('./public')
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  const uri = request.url;
+  const reqMeth = request.method;
+console.log(uri)
```

Comment:
Your need to declare your function outside of connection callback, so that it is not recreated every-time a request is made.